deployment/helm-k8s/templates/response/ingress.yaml

```diff
@@ -0,0 +1,41 @@
+{{- if .Values.response.enabled -}}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ include "siembol.response.fullname" $ }}
+  namespace: {{ .Values.namespace }}
+  annotations:
+    cert-manager.io/issuer: ca-issuer
+spec:
+  {{ if .Values.tls.enabled -}}
+  tls:
+    - hosts:
+      - {{ .Values.response.ingress.hostname }}
+      secretName: {{ .Values.response.appName }}-tls
+  {{ end }}
+  rules:
+    - host: {{ .Values.response.ingress.hostname }}
+      http:
+        paths:
+        - path: /health
+          pathType: Prefix
+          backend:
+            service:
+              name: {{ include "siembol.response.fullname" $ }}
+              port:
+                number: {{ .Values.response.service.port }}
+        - path: /info
+          pathType: Prefix
+          backend:
+            service:
+              name: {{ include "siembol.response.fullname" $ }}
+              port:
+                number: {{ .Values.response.service.port }}
+        - path: /api
```

Comment:
do we want to expose always? since we are using disabled authentication it should be exposed only in dev mode along with swagger

Reply:
added flag